samples/ResourceOwnerPasswordFlow/.editorconfig

```diff
@@ -0,0 +1,24 @@
+# EditorConfig is awesome:
```

Comment:
Is EditorConfig a popular editor? I don't mind merging this config file it that's really the case :smile: